content/spin/spin-oci.md

```diff
@@ -17,21 +18,29 @@ url = "https://github.com/fermyon/developer/blob/main//content/spin/spin-oci.md"
 
 ## Spin Open Container Initiative (OCI) Support
 
-From [Spin v0.8.0](https://www.fermyon.com/blog/spin-v08) onwards, you can store and run Spin applications stored in an OCI compatible registry. Spin developers now have a powerful workflow that enables them to use idiomatic infrastructure tooling, such as [GitHub Container Registry (GHCR)](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) or [DockerHub](https://hub.docker.com/), to store, deploy, and share their Spin applications. 
+With Spin's Open Container Initiative (OCI) support, you can package and save your Spin application as an OCI artifact in a registry like [GitHub Container Registry (GHCR)](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) or [DockerHub](https://hub.docker.com/).
 
 ## Prerequisites
 
-First, follow [this guide](./install.md) to install Spin v0.8.0 or later. To ensure you have the correct version, you can check with this command:
+First, follow [this guide](https://developer.fermyon.com/spin/install) to ensure you have the latest version of Spin installed. You can check the Spin version using the following command:
 
 <!-- @selectiveCpy -->
 
 ```bash
 $ spin --version
 ```
 
+## Building and Running Spin Applications With Docker (Video)
+
+The following video shows you how to push a Spin app to GHCR, and then run that artifact with Docker. The video also contains additional information about signing and verifying your GHCR artifacts.
```

Comment:
The repeated mentions of Docker were confusing to me because the written "tutorial" never mentions Docker again (and the first part of the video is all Spin, no Docker).  Suggest changing the title to "Publishing and Running Spin Applications Using Registries" and the first sentence to "...run that artifact with Spin or with Docker."

Reply:
Thanks, perfect!